service/src/main/resources/transactions_dump.txt

```diff
@@ -0,0 +1,27 @@
+bookingStatus|endToEndId|mandateId|creditorId|bookingDate|valueDate|currency|amount|creditorName|creditorIban|currency|ultimateCreditor|debtorName|debtorIban|currency|ultimateDebtor|remittanceInformationUnstructured|remittanceInformationStructured|purposeCode|bankTransactionCode|proprietaryBankTransactionCode
```

Comment:
Replace content with sample data and rename to *.csv?

Reply:
done